### PR TITLE
Release v2.19.1

### PR DESCRIPTION
**Pull request by bengourley**
Bumps both bugsnag-android and bugsnag-cocoa to latest versions.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,20 @@
 Changelog
 =========
 
+## 2.19.1 (2019-05-21)
+
+### Enhancements
```
Comment (kattrali) [inline CHANGELOG.md]:
```suggestion
### Bug fixes
```

Since its just fixes

**Comment by AleksandrChernyavenko** [comment]
It would be nice to have new release with this PR

**Comment by bengourley** [comment]
@AleksandrChernyavenko sorry for the delay, we had to wait for further end to end tests (that are not on CI) to complete.

This has now shipped.

**Comment by AleksandrChernyavenko** [comment]
@bengourley Thank you!

**Pull request merged**